saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentDetails.jsp

```diff
@@ -184,13 +184,40 @@ $(document).ready(function() {
 			</ul>
 		</c:if>
 	
-
-		<c:if test="${!empty judgment.sourceInfo}">
-			<h4><spring:message code="judgment.sourceInfo" />:</h4>
-			<a href="${judgment.sourceInfo.sourceJudgmentUrl}" rel="nofollow" >
-				<c:out value="${judgment.sourceInfo.sourceJudgmentUrl}" />
-			</a>
+		
+		<c:if test="${!empty judgment.referencedCourtCases}">
+		
+			<div class="panel panel-default panel-generated" >
+				<spring:message code="judgmentDetails.enrichmentTag.header" var="enrichmentTagHeader" />
+				<spring:message code="judgmentDetails.enrichmentTag.hintText" var="enrichmentTagHintText" />
+				<div class="panel-heading"><spring:message code="judgmentDetails.enrichmentTag.header" />:<saos:hint title="${enrichmentTagHeader}" content="${enrichmentTagHintText}" /></div>
+				<div class="panel-body">
+					<div class="label-title"><spring:message code="judgmentDetails.enrichmentTag.referencedCourtCases" />:</div>
+					<div class="desc">
+					
+						<c:forEach items="${judgment.referencedCourtCases}" var="refCourtCase" >
+							<p>
+								<c:choose>
+									<c:when test="${!empty refCourtCase.judgmentIds}" >
+										<c:forEach items="${refCourtCase.judgmentIds}" var="judgmentId" >
+											<a href="/saos-webapp/judgments/${judgmentId}">
```

Comment:
You can't add link with context path '/saos-webapp' hardcoded. This will not work if we change it.